cre.cpp

```diff
@@ -3318,6 +3318,14 @@ static int regularizeRegisteredFontsWeights(lua_State *L) {
 	return 0;
 }
 
+static int checkRegex(lua_State *L) {
+	CreDocument *doc		= (CreDocument*) luaL_checkudata(L, 1, "credocument");
+	const char *l_pattern   = luaL_checkstring(L, 2);
+	lString32 pattern		= lString32(l_pattern);
+    lua_pushinteger(L, checkRegex(pattern));
```

Comment:
Some mixed indentations in there.
Do you think your checkRegex might return other values than 0 or 1? If no, better make this returns a boolean. If yes, well, ok :)

Reply:
I would leave an int here. I would lik a more verbose error message here (we get that from the exception ).